great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -37,11 +37,11 @@ class GeCloudStoreBackend(StoreBackend, metaclass=ABCMeta):
     def __init__(
         self,
         ge_cloud_credentials: Dict,
-        ge_cloud_base_url: Optional[str] = "https://app.greatexpectations.io/",
```

Comment:
I wonder if there is a case for this being None

Reply:
Looked through the file; we should consider asserting that this is a proper URL in the constructor based on how it is used in list_keys

Reply:
agree, we can make this change in another PR